custom_components/tuya_local/devices/zg205w_mmWave_presence_sensor.yaml

```diff
@@ -177,11 +171,133 @@ secondary_entities:
         range:
           min: 1
           max: 30
-  - entity: sensor
-    class: illuminance
+  - entity: number
+    name: Motion minimum detection distance
+    category: config
+    icon: mdi:signal-distance-variant
+    hidden: true
     dps:
-      - id: 101
+      - id: 110
+        type: integer
+        name: value
+        unit: m
+        range:
+          min: 0
+          max: 1000
+        mapping:
+          - scale: 100
+            step: 10
+  - entity: number
+    name: Small motion minimum detection distance
+    category: config
+    icon: mdi:signal-distance-variant
+    hidden: true
+    dps:
+      - id: 111
+        type: integer
+        name: value
+        unit: m
+        range:
+          min: 0
+          max: 1000
+        mapping:
+          - scale: 100
+            step: 10
+  - entity: number
+    name: Breathing minimum detection distance
+    category: config
+    icon: mdi:signal-distance-variant
+    hidden: true
+    dps:
+      - id: 112
+        type: integer
+        name: value
+        unit: m
+        range:
+          min: 0
+          max: 1000
+        mapping:
+          - scale: 100
+            step: 10
+  - entity: number
+    name: Adaptive testing time
+    category: diagnostic
+    icon: mdi:timer
+    hidden: true
+    dps:
+      - id: 113
         type: integer
+        name: value
+        unit: s
+        range:
+          min: 0
+          max: 300
+  - entity: switch
+    name: Adaptive test duration
+    category: diagnostic
+    icon: mdi:test-tube
+    dps:
+      - id: 114
+        name: switch
+        type: boolean
+  - entity: button
+    name: Factory reset
+    category: diagnostic
+    icon: mdi:restore-alert
+    dps:
+      - id: 115
+        type: boolean
+        name: button
+        optional: true
+  - entity: sensor
+    name: Movement state
+    class: enum
+    icon: mdi:cursor-move
+    dps:
+      - id: 116
+        type: string
         name: sensor
-        unit: lx
-        class: measurement
+        mapping:
+          - dps_val: large_move
+            value: Large
+          - dps_val: small_move
+            value: Small
+          - dps_val: breathe
+            value: Breathe
+          - dps_val: none
+            value: None
+  - entity: switch
+    name: Small motion self-test
+    category: diagnostic
```

Comment:
Switch should be config, not diagnostic. Would this be better as a button? (ie, when the test is finished, does this automatically return to false?)